pkg/placeholders/utils.go

```diff
@@ -120,3 +124,21 @@ func InterpolatePlaceholders(source string) string {
 		}
 	})
 }
+
+// GetBodyFromFileOrInlined returns the correct content for the body of a request.
+// the body of the request can either be inlined, or come from a file
+func GetBodyFromFileOrInlined(source string) (*string, error) {
```

Comment:
If we can add a unit test for this method that'd be great.

Reply:
done